pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -490,6 +490,41 @@ def test_aggregate_subsampling_profile_compress():
     pd.testing.assert_frame_equal(result, expected_result)
 
 
+def test_aggregate_subsampling_profile_compress_multiple_queries():
+    compress_file = os.path.join(tmpdir, "test_aggregate_compress.csv.gz")
+
+    _ = ap_subsample.aggregate_profiles(
+        output_file=compress_file,
+        compute_subsample=True,
+        compression_options={"method": "gzip"},
+        n_strata=1,  # this will force multiple queries from each compartment
```

Comment:
can you run this test multiple times looping through different `n_strata`:

E.g.

```python
for n_strata in [1, 2, 3]:
    ...
   aggregate_profiles(n_strata=n_strata)
```

and confirm that the expected result doesn't change

Reply:
can you also add a separate test where you set `n_strata` to an invalid argument?

Reply:
Yes this is a good idea in terms of testing multiple values for `n_strata`.

We can test invalid `n_strata` values... but the only truly invalid values should be `< 1`, and there is an explicit assertion for that.  But we can still include a test.